Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM python:3.6
-ENV PYTHONUNBUFFERED=1 \
-    POSTGRES_PASSWORD='password'
+
+RUN rm -rf /accounting /venvs
```

Comment:
We are using docker? That's different from Ocim's use of Vragrant?

Reply:
This is just for the development environment, and I don't think that's an issue -- you just have to run `make dev.up` and it's done :) I don't plan to deploy with Docker though.

We can also put a Vagrantfile to work alongside Docker, but I found Docker easier/faster to get started with.

Reply:
@UmanShahzad I won't block on this, but we were also targeting having the same environment as for Ocim, so I would have expected this to be discussed at least. Are there other differences between the two, aside from the dependencies versions which would be more recent here?

Reply:
I don't think there are other differences, no. But it's possible I'm forgetting something.

But this is a development environment thing, and I would say it should be up to the developer's discretion to choose one? I don't know how `vagrant up` vs. `make dev.up` affects the actual product -- except that the latter allows you to work faster on it.